tests/unit/ProcessMaker/Models/ProcessCategoryTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace ProcessMaker\Models;
+
+use ProcessMaker\Models\ProcessCategory;
+use Tests\TestCase;
+
+class ProcessCategoryTest extends TestCase
+{
+    /**
+     * Test ScriptCategory Model.
+     *
+     * @return void
+     */
+    public function testGetNamesByIds()
```

Comment:
Please in order to create more categories you can use:
```
$category = ProcessCategory::factory()->create();
$category->name;
        $category->id;
```

Reply:
Thanks, suggestion taken and applied.